pkg/filters/builder/builder.go

```diff
@@ -76,7 +76,7 @@ func (b *Builder) build(data map[string]interface{}, v interface{}) error {
 		return err
 	}
 
-	return codectool.Decode(&result, v)
+	return yaml.Unmarshal(result.Bytes(), v)
```

Comment:
It's consistent to use `codecool.UnmarshalYAML` in case of replacing the vendor in one time and one place.

Reply:
ok, I will switch to that function.